### PR TITLE
profile person

### PR DESCRIPTION
**Pull request by Mateusszzp**

### CODE REVIEW

File: src/Features/Movies/SingleMovie/styled.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import styled from "styled-components";
 
-export const SingleMovieContainer = styled.main`
+export const SingleMovieContainer = styled.main`//
```
Comment (KubaTuta) [inline src/Features/Movies/SingleMovie/styled.js]: empty comment?

File: src/common/DetailsTileStyles/styled.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ export const PosterWrapper = styled.div`
 `;
 
 export const Poster = styled.img`
-    width: 100%;
+    //width: 100%;
```
Comment (KubaTuta) [inline src/common/DetailsTileStyles/styled.js]: forget to remove wheter forget to uncomment?

**Review by KubaTuta — APPROVED** [review]
awesome job

File: src/Features/Movies/SingleMovie/styled.js (hunk 1 above)
Comment (Mateusszzp) [reply src/Features/Movies/SingleMovie/styled.js]: it's Hania's component I didn't want to change it but I marked I don't know what her plans are

File: src/common/DetailsTileStyles/styled.js (hunk 2 above)
Comment (Mateusszzp) [reply src/common/DetailsTileStyles/styled.js]: ok i am correcting it now

**Pull request merged**